README.md

```diff
@@ -124,6 +135,18 @@ message DisableVolumeReplicationResponse {
 }
 ```
 
+#### Error Scheme
+
+| Condition | gRPC Code | Description | Recovery Behavior |
+|-------------------|-------------------|-------------------|-------------------|
+| Missing required field | INVALID_ARGUMENT | Indicates that a required field is missing from the request. | Caller MUST fix the request by adding the missing required field before retrying. |
+| Source does not exist | NOT_FOUND | Indicates that the specified source does not exist. | Caller MUST verify that the source is correct, the source is accessible, and has not been deleted before retrying with exponential back off. |
+| Volume in use | FAILED_PRECONDITION | Indicates that the volume corresponding to the specified `volume_id` could not be Disabled due to failed precondition (for example volume is not enabled). | Caller SHOULD ensure that volume is enabled. |
```

Comment:
if the mirroring is not enabled we should return success as spec need to be idempotent

Reply:
yeah right, what can be precondition here?

Reply:
no precondition. we can remove the pre-condition here

Reply:
missed to remove precondiftion here?